src/App.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import SignInForm from './components/Deviseforms/signin';
+import SignUpForm from './components/Deviseforms/signup';
+import Homepage from './components/Homepage/homepage';
+import Splash from './components/Shared/splash';
+import Additempage from './components/Additempage/Additemcomponent';
+import Sidebar from './components/Shared/sidebar';
+import CarDetailsPage from './components/CarDetails/cardetailspage';
+import Reservepage from './components/Reservepage/reservecomponent';
+import MyReservepage from './components/MyReservations/myreservepage';
+import DeletePage from './components/forms/DeletePage';
+
+const App = () => (
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const Additempage = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 6):

  
![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/9a5591c4-008a-468b-b0e2-82b4f159eb68)


That way, your code will meet the latest standards. 💪